README.md

```diff
@@ -1,2 +1,19 @@
 # TIDES
-Transit ITS Data Exchange Specification for historical ITS data
+
+Transit ITS Data Exchange Specification (TIDES) for historical transit operations data.
+
+This repository provides data schemas and tools to support the access, management, and improvement of historical transit operations data, including vehicle operations, passenger activity, fare collection, and other similar data.
+
+### TIDES specification
+The TIDES specification is maintained in the `/spec` sub-directory as a series of JSON tables.
+
+Data Table schemas
+Data table schemas are specified in JSON and are compatible with the [frictionless data](https://specs.frictionlessdata.io/table-schema/) table schema standards.
+
```

Comment:
Add the following section here:

## Contributing to TIDES

Those who want to help with the development of the TIDES specification should review the guidance in the [CONTRIBUTING.md][…link…] file.